MobileCenterPush/MobileCenterPush.xcodeproj/project.pbxproj

```diff
@@ -271,6 +271,7 @@
 				049BC81E1ECE333700FB6719 /* iOS.xcconfig */,
 				D3AD5F3E1E5D69B0001627AB /* iOS.modulemap */,
 			);
+			name = Support;
```

Comment:
Can you remove this line? I might be wrong but this one isn't necessary because `Support` is a folder and the group is referencing the folder. The name will use the folder name automatically.